test/ufuzz.js

```diff
@@ -183,43 +284,57 @@ function log() {
     console.log(beautify_result);
     console.log("uglified result:");
     console.log(uglify_result);
+    if (!ok) console.log("!!!!!! Failed...");
 }
 
 var num_iterations = +process.argv[2] || 1/0;
-var verbose = !!process.argv[3];
+var verbose = process.argv[3] === 'v' || process.argv[2] === 'v';
+var verbose_interval = process.argv[3] === 'V' || process.argv[2] === 'V';
```

Comment:
Just keep this simple and make specifying `num_iterations` mandatory when you need to specify a verbose mode.